build/allowedValues.json

```diff
@@ -12,6 +12,87 @@
   "numberOfDataNodes" : 50,
   "numberOfClientNodes" : 20,
   "diskSizes" : ["Small", "Medium", "Large", "XLarge", "XXLarge"],
+  "vmAcceleratedNetworking": [
```

Comment:
It's sad we need to maintain these compat lists rather than `ARM` exposing this in a apply or noop fashion.

Reply:
Agreed 😢 ours can't be only the template that does this